shareledger/chain.json

```diff
@@ -0,0 +1,160 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "shareledger",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.sharering.network/",
+  "pretty_name": "Shareledger",
+  "chain_id": "ShareRing-VoyagerNet",
+  "bech32_prefix": "shareledger",
```

Comment:
Sorry, I missed one more.
Can you please register this prefix to SLIP-0173?
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done, just waiting for them to approve the pull request.